ios/Framework/Core/Base/BaseScreenlet.swift

```diff
@@ -152,16 +152,6 @@ import QuartzCore
 		let view = createScreenletViewFromNib()
 
 		if let viewValue = view {
-			//FIXME: full-autoresize value. Extract from UIViewAutoresizing
```

Comment:
look @feliperp-81 
The problem with autolayout is finally fixed

Thanks for finding it!